apollo-portal/src/test/java/com/ctrip/framework/apollo/openapi/filter/ConsumerAuthenticationFilterTest.java

```diff
@@ -89,4 +108,99 @@ public void testAuthFailed() throws Exception {
     verify(consumerAuditUtil, never()).audit(eq(request), anyLong());
     verify(filterChain, never()).doFilter(request, response);
   }
+
+
+  @Test
+  public void testRateLimitSuccessfully() throws Exception {
+    String someToken = "someToken";
+    Long someConsumerId = 1L;
+    int qps = 5;
+    int durationInSeconds = 10;
```

Comment:
10 seconds is too long for a unit test to finish, it's better to shorten the running time.

Reply:
OK, I'll shorten it to 3 seconds.